src/api/rest/server/mcis/control.go

```diff
@@ -178,6 +178,19 @@ func RestGetAllMcis(c echo.Context) error {
 		content.Mcis = result
 		common.PrintJsonPretty(content)
 		return c.JSON(http.StatusOK, &content)
+	} else if option == "simple" {
+		// mcis in detail (with status information)
+		detail := "simple"
+
+		result, err := mcis.CoreGetAllMcis(nsId, detail)
```

Comment:
`detail` 변수를 새로 만들지 않고 다음과 같이 할 수도 있긴 한데.. minor 한 사항이기는 합니다.. 😊

```suggestion
	} else if option == "simple" {
		// mcis in detail (with status information)
		result, err := mcis.CoreGetAllMcis(nsId, option)
```

Reply:
좋네요.. ^^ 업데이트 하겠습니다.